src/main/java/nextstep/subway/path/application/ShortestPathFinder.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.subway.path.application;
+
+import nextstep.subway.line.domain.Sections;
+import nextstep.subway.path.domain.Path;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.Stations;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+public class ShortestPathFinder {
+
+    public static Path findShortestPath(
+        Sections sections, Stations stations, Station sourceStation, Station targetStation) {
+        WeightedMultigraph<Station, DefaultWeightedEdge> graph = new WeightedMultigraph<>(DefaultWeightedEdge.class);
+
+        stations.forEach(graph::addVertex);
+        sections.forEach(section -> graph.setEdgeWeight(
```

Comment:
```suggestion
        sections.forEach(section -> {
            DefaultWeightedEdge edge = graph.addEdge(section.getUpStation(), section.getDownStation());
            graph.setEdgeWeight(edge, section.getDistance());
        });
```
이 경우엔 코드를 한줄에 다 작성하기 보단 읽기 좋게 변수로 나눠주는걸 추천합니다